test/evergreen.yml

```diff
@@ -1586,6 +1586,27 @@ tasks:
           display_name: Coverage report
           remote_file: wiredtiger/${build_variant}/${revision}/coverage_report/coverage_report_${build_id}.html
 
+  - name: test-wtperf
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          posix_configure_flags: --enable-strict --enable-diagnostic
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix"
+          script: |
+            set -o errexit
+            set -o verbose
+
+            mkdir WT_TEST
```

Comment:
I know Python test suite will generate `WT_TEST` directory automatically, not sure about wtperf. We could save this line (though a comment would be helpful) if it's the case.

Reply:
Ok. Let me confirm and then remove the line if needed.